ocrd/ocrd/resource_manager.py

```diff
@@ -12,7 +12,6 @@
 
 from ocrd_validators import OcrdResourceListValidator
 from ocrd_utils import getLogger
-from ocrd_utils.constants import HOME, XDG_DATA_HOME, XDG_CONFIG_HOME
```

Comment:
I am surprised to see you move imports from module level to functions here. What's the reason for this?

Reply:
Ah, apparently to be able to move away the resource config after `ocrd.resource_manager` has been loaded already in your new `test_resource_manager.py`. 

But couldn't you achieve the same result by moving L9 `from ocrd.resource_manager import OcrdResourceManager` to the test fixture instead in the test case itself?

Reply:
I tried that, i.e. only instantiating `OcrdResourceManager` in the `monkey_patch_temp_xdg` method but that will only work once, because top-level import only happens once. Only solution I see that doesn't involve doing method-level imports in `OcrdResourceManager` is to squish all the tests into a single method.

Reply:
31505067